server_test.go

```diff
@@ -23,6 +23,7 @@ func TestServer(t *testing.T) {
 	defer report()
 
 	loggerFactory := logging.NewDefaultLoggerFactory()
+	// loggerFactory.DefaultLogLevel = logging.LogLevelTrace
```

Comment:
Can we remove the dead code?

Reply:
Thanks! Fixed per [f3af4ad](https://github.com/pion/turn/pull/267/commits/f3af4ad1a272d8a2a46a8170b983a3641e6d1dfc).